manifest-generation/config-from-cf-internal.yml

```diff
@@ -85,6 +92,10 @@ properties:
     tls:
       client_cert: (( merge ))
       client_key: (( merge ))
+    etcd:
+      client_cert: (( properties.metron_agent.etcd.client_cert ))
```

Comment:
This reference and the following line seem circular: shouldn't they be `null` as well?